Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volumegroup-snapshot.md

```diff
@@ -0,0 +1,108 @@
+---
+title: VolumeGroup Snapshots
+---
+
+A volume group snapshot represents “copies” from multiple volumes that are taken at the same point-in-time.
+It can be used either to rehydrate new volumes (pre-populated with the snapshot data) or
+to restore existing volumes to a previous state (represented by the snapshots).
+
+
+## Prerequisites
+
+- Install the [snapshot controller and snapshot v1 CRD](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage),
```

Comment:
`snapshot controller and snapshot v1 CRD` to `snapshot controller and snapshot and volume group snapshot CRD`